xprof/xprof.rb.in

```diff
@@ -883,20 +896,23 @@ def all_env_tracers
 
     # Find and Save the original lib path
     libenv = "LTTNG_UST_#{name.upcase}_#{lib.upcase}"
+    libso = "#{lib}.so"
     if (e = env_fetch_first(libenv))
       LOGGER.warn("#{libenv} was already set, will use this path #{e} for the #{name} loader")
       # TODO: Verify that this guy exist?
-    elsif (libpath = whichlib64("#{lib}.so"))
+    elsif (libpath = whichlib64(libso))
+      h[libenv] = libpath
+    elsif (libpath = dlinfo_wrapper(libso, MPITopo.master?))
```

Comment:
Hum maybe remove the `verbose` mode for dlinfro_wrapper call here? If we want to be cleaner, we will need to use all the `info::DEBUG`  and stuff. If Not I'm afraid that we will spammed by a lot of `stderr` call.  (Indeed will not find levelzero, and stuff on nvidia so dlopen will fail, but that ok it doesn't matter by default)